### PR TITLE
Rename cairo-rs to cairo-vm

### PR DESCRIPTION
**Pull request by pefontana**
# Rename cairo-rs to cairo-vm

## Checklist
- [ ] Linked to Github Issue
- [ ] Unit tests added
- [ ] Integration tests added.
- [ ] This change requires new documentation.
  - [ ] Documentation has been added/updated.
  - [ ] CHANGELOG has been updated.

### CODE REVIEW

File: docs/hint_processor/hint_design.md [hunk 1]
```diff
@@ -67,11 +67,11 @@ Notes
 Notes
 - Learn to use wasm tooling
 - Learn how to compile python to wasm
-- Definir the interfaces between the embedded wasm vm and cairo-rs
+- Definir the interfaces between the embedded wasm vm and cairo-vm
```
Comment (Oppen) [inline docs/hint_processor/hint_design.md:70]: Define?

**Review by Oppen — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,15 +1,15 @@
 <div align="center">
 <img src="./bonaparte.webp" height="150">
 
-### ⚡ Cairo-rs ⚡
+### ⚡ Cairo-vm ⚡
 
 A faster and safer implementation of the Cairo VM in Rust
 
-[Report Bug](https://github.com/lambdaclass/cairo-rs/issues/new?labels=bug&title=bug%3A+) · [Request Feature](https://github.com/lambdaclass/cairo-rs/issues/new?labels=enhancement&title=feat%3A+)
+[Report Bug](https://github.com/lambdaclass/cairo-vm/issues/new?labels=bug&title=bug%3A+) · [Request Feature](https://github.com/lambdaclass/cairo-vm/issues/new?labels=enhancement&title=feat%3A+)
```
Comment (MegaRedHand) [inline README.md:8]: We should change this link to use the bug issue template:
```
https://github.com/lambdaclass/cairo-vm/issues/new?assignees=&labels=bug&projects=&template=bug_report.md&title=bug%3A+
```

**Review by MegaRedHand — APPROVED** [review]

**Comment by MegaRedHand** [comment]
Should we mention this in the changelog?

**Comment by Oppen** [comment]
> Should we mention this change in the changelog?

Maybe. Since some users point to specific revisions, this is probably true. In general, tho, you'd refer to the crate on crates.io, which has always been `cairo-vm`.

File: README.md (hunk 2 above)
Comment (pefontana) [reply README.md]: good!
done

File: docs/hint_processor/hint_design.md (hunk 1 above)
Comment (pefontana) [reply docs/hint_processor/hint_design.md]: jeje
done

File: vm/Cargo.toml [hunk 3]
```diff
@@ -1,8 +1,11 @@
 [package]
 name = "cairo-vm"
 description = "Blazing fast Cairo interpreter"
-repository = "https://github.com/lambdaclass/cairo-vm/"
-readme= "../README.md"
+version.workspace = true
+edition.workspace = true
+license.workspace = true
+repository.workspace = true
+readme.workspace = true
```
Comment (Oppen) [inline vm/Cargo.toml]: I thought this was already merged.

**Review by Oppen — APPROVED** [review]

File: vm/Cargo.toml (hunk 3 above)
Comment (Oppen) [reply vm/Cargo.toml]: Oh it was part of resolving conflicts.

**Review by juanbono — APPROVED** [review]

**Comment by pefontana** [comment]
> > Should we mention this change in the changelog?
> 
> Maybe. Since some users point to specific revisions, this is probably true. In general, tho, you'd refer to the crate on crates.io, which has always been `cairo-vm`.

I can add it :why-not

**Pull request merged**